dev-packages/node-integration-tests/suites/tracing-experimental/mysql/scenario-withoutCallback.js

```diff
@@ -24,7 +24,7 @@ connection.connect(function (err) {
   }
 });
 
-Sentry.startSpan(
+Sentry.startSpanManual(
```

Comment:
LOL, this test only passed "on accident" before (but this shows we are being more "strict" now than we used to).